go-controller/pkg/ovn/master.go

```diff
@@ -218,32 +218,10 @@ func (oc *Controller) upgradeOVNTopology(existingNodes *kapi.NodeList) error {
 func (oc *Controller) StartClusterMaster() error {
 	klog.Infof("Starting cluster master")
 
-	metrics.RegisterMasterPerformance(oc.nbClient)
+	metrics.RegisterMasterPerformance(oc.mc.nbClient)
 	metrics.RegisterMasterFunctional()
-	metrics.RunTimestamp(oc.stopChan, oc.sbClient, oc.nbClient)
-	metrics.MonitorIPSec(oc.nbClient)
-	if config.Metrics.EnableConfigDuration {
-		// with k=10,
-		//  for a cluster with 10 nodes, measurement of 1 in every 100 requests
-		//  for a cluster with 100 nodes, measurement of 1 in every 1000 requests
-		metrics.GetConfigDurationRecorder().Run(oc.nbClient, oc.kube, 10, time.Second*5, oc.stopChan)
-	}
-	oc.podRecorder.Run(oc.sbClient, oc.stopChan)
-
-	// enableOVNLogicalDataPathGroups sets an OVN flag to enable logical datapath
-	// groups on OVN 20.12 and later. The option is ignored if OVN doesn't
-	// understand it. Logical datapath groups reduce the size of the southbound
-	// database in large clusters. ovn-controllers should be upgraded to a version
-	// that supports them before the option is turned on by the master.
-	nbGlobal := nbdb.NBGlobal{
-		Options: map[string]string{"use_logical_dp_groups": "true"},
-	}
-	if err := libovsdbops.UpdateNBGlobalSetOptions(oc.nbClient, &nbGlobal); err != nil {
-		klog.Errorf("Failed to set NB global option to enable logical datapath groups: %v", err)
-		return err
-	}
```

Comment:
I'm wondering where did this go 0_o ; is this code no longer required ? 

Any chance we can isolate this into a separate commit to make more sense of it ?

Reply:
@maiqueb this code goes to https://github.com/ovn-org/ovn-kubernetes/blob/204a3897a2ee523403e3a58f99d23ecb546048e7/go-controller/pkg/ovn/ovn.go#L302. We'd need it to move before controller.StartClusterMaster() because logically they are needed for all controllers. I don't think we can isolate it into separate commit?

Please suggest